config/config.example.yml

```diff
@@ -21,6 +21,7 @@
 # This is the domain of the main public website
 # It is used by the view to generate links back to the rest of the site on the login form.
 public_site_domain: beyondz.org
+cookie_domain: .beyondz.org # this should be the *top* domain - so even for staging, it should be .beyondz.org still
```

Comment:
Can you just document what the ```cookie_domain``` controls.  Similar to how the above comment about ```public_site_domain``` tells what the config value controls?